src/ZipkinOpenTracing/Scope.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ZipkinOpenTracing;
+
+use OpenTracing\Scope as OTScope;
+
+class Scope implements OTScope
+{
+    public function __construct($scopeManager, $wrapped)
+    {
+        $this->scopeManager = $scopeManager;
```

Comment:
Can we declare attributes? and final the class to keep consistency with the rest of the library?